app/src/androidTest/java/ch/epfl/sweng/erpa/GameListTest.java

```diff
@@ -0,0 +1,90 @@
+package ch.epfl.sweng.erpa;
+
+import android.support.test.espresso.intent.rule.IntentsTestRule;
+import android.support.test.espresso.matcher.ViewMatchers;
+import android.support.test.runner.AndroidJUnit4;
+import android.support.v7.widget.CardView;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static android.support.test.espresso.Espresso.onView;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(AndroidJUnit4.class)
+public class GameListTest {
+    @Rule
+    public final IntentsTestRule<GameList> intentsTestRule = new IntentsTestRule<>(GameList.class);
+
+    @Test
+    public void testMinNumberOfCardsDisplayed() {
+        RecyclerView view = intentsTestRule.getActivity().findViewById(R.id.recyclerView);
+        int itemCount = view.getLayoutManager().getItemCount();
+
+        assertTrue(itemCount >= 5);
+    }
+
+    @Test
+    public void testMaxNumberOfCardsDisplayed() {
+        RecyclerView view = intentsTestRule.getActivity().findViewById(R.id.recyclerView);
+        int itemCount = view.getLayoutManager().getItemCount();
+
+        assertTrue(itemCount <= 25);
```

Comment:
Where is this magic value coming from?

Reply:
It's a magic value I picked for testing as the real thing isn't implemented yet (still waiting on #41 )

Reply:
Can you please add a comment on what this number means and how to replace once #41 is implemented?